bay/plugins/mounts.py

```diff
@@ -25,26 +25,43 @@ def load(self):
 
 
 @click.command()
+@click.option("--profile-only", "-p", default=False, is_flag=True)
 @click.pass_obj
-def mounts(app):
+def mounts(app, profile_only):
     """
     List all current dev mounts.
     """
     dev_mounts = defaultdict(dict)
+
     for container in app.containers:
         unmounted_devmodes = set(container.devmodes.keys())
         runtime_options = app.containers.options(container)
         if runtime_options:
             devmodes = app.containers.options(container).get('devmodes')
             dev_mounts[container.name]['mounted'] = sorted(devmodes)
             dev_mounts[container.name]['unmounted'] = unmounted_devmodes.difference(devmodes)
+        else:
+            if profile_only:
+                # either no devmodes, or not in profile
+                continue
+            dev_mounts[container.name]['mounted'] = []
+            dev_mounts[container.name]['unmounted'] = unmounted_devmodes
 
     for name, devmodes in dev_mounts.items():
-        click.echo('{}: \nMounted: {}\nUnmounted: {}'.format(
-            CYAN(name),
-            GREEN(', '.join(sorted(devmodes['mounted']))),
-            PURPLE(', '.join(sorted(devmodes['unmounted']))),
-        ))
+        if not devmodes['mounted']:
```

Comment:
@andrewgodwin well, depends if you do want to print container name at all when it has no devmodes at all?